spec/features/users/current_user_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+require "./app/models/user"
```

Comment:
You don't need to require the user file manually. Since we are calling "rails_helper", it loads the whole rails environment for us.